docset/windows/scheduledtasks/Disable-ScheduledTask.md

```diff
@@ -129,14 +129,14 @@ Accept wildcard characters: False
 ```
 
 ### -TaskPath
-Specifies the path for a scheduled task in Task Scheduler namespace.
-You can use **\** for the root folder.
+Specifies an array of one or more paths for scheduled tasks in Task Scheduler namespace. You can use **"*"** for a wildcard character query.
+You can use **\\*** for the root folder. To specify a full TaskPath you need to include the leading and trailing **\**.
```

Comment:
```suggestion
You can use **\\*** for the root folder. To specify a full TaskPath you need to include the leading and trailing **\\**.
```
I think you should fix this in all the cmdlets.

Reply:
Thanks, missed the escaping \ :(